spring-cloud-gcp-pubsub-stream-binder/README.adoc

```diff
@@ -0,0 +1,57 @@
+= Spring Cloud GCP Stream for Pub/Sub
+
+This module contains the Spring Cloud GCP Stream for Pub/Sub logic.
+
+== Spring Cloud GCP Stream Config
+
+
+Maven coordinates, using Spring Cloud GCP BOM:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-cloud-gcp-pubsub-stream-binder</artifactId>
+</dependency>
+----
+
+Gradle coordinates:
+
+[source]
+----
+dependencies {
+    compile group: 'org.springframework.cloud', name: 'spring-cloud-gcp-pubsub-stream-binder'
+}
+----
+
+=== Setup for Pub/Sub Emulator
+
+To run this binder with local emulator you just need to provide the property `PUBSUB_EMULATOR_HOST` as specified by [``= Spring Cloud GCP Stream for Pub/Sub
```

Comment:
[``= Spring Cloud GCP Stream for Pub/Sub

Seems like this is weirdly formatted?